src/app/workflow/launch-third-party/launch-third-party.component.ts

```diff
@@ -166,7 +166,7 @@ export class LaunchThirdPartyComponent extends Base implements OnChanges, OnInit
 
   bdCatalystSevenBridgesTooltip$: Observable<string> = combineLatest([this.hasContent$, this.hasHttpImports$]).pipe(
     map(([hasContent, hasHttpImports]) =>
-      this.sevenBridgesTooltip(hasContent, hasHttpImports, 'NHLBI BioData Catalyst powered by Seven Bridges')
+      this.sevenBridgesTooltip(hasContent, hasHttpImports, 'BioData Catalyst (BDC) Powered by Seven Bridges')
```

Comment:
Also updated some language:
![image](https://user-images.githubusercontent.com/97123241/214366972-b91edf26-1ff2-4827-adb8-c748ba45524c.png)
To match:
![image](https://user-images.githubusercontent.com/97123241/214367268-9f904c20-5247-465c-a36f-3b176f3264e5.png)

Reply:
Super nit: I prefer a lower case "p" in `powered`; but if it's in the guidelines...